src/libertem/web/events.py

```diff
@@ -17,6 +16,9 @@ def add_handler(self, handler):
     def remove_handler(self, handler):
         self.handlers.remove(handler)
 
+    def get_number_of_handlers(self):
```

Comment:
I add this to see if we have connections to the server through the current session ID or not. This is to handle deleting the sessions once the user is done working.